src/components/Book.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
-function Book() {
+function Book(book) {
+  let { title, Author } = book;
+  title = 'Mistress of Spices';
+  Author = 'Divakaruni';
   return (
     <div>
-      <h1>Machine learning by Chime</h1>
+      <h1>{title}</h1>
+      <p>{Author}</p>
     </div>
   );
 }
```

Comment:
- [ ] Please, consider adding the button for removing the book, as it was requested by me before (it is a requirement of the project) I know you can do it easily 👍 Consider also that for this milestone the button will not have any functionality.